optuna/importance/_base.py

```diff
@@ -63,43 +73,91 @@ def evaluate(
         raise NotImplementedError
 
 
-def _get_distributions(study: Study, params: Optional[List[str]]) -> Dict[str, BaseDistribution]:
+def _get_info_distribution(distribution: BaseDistribution) -> Dict[str, Any]:
+    info_distribution = {}
+    if isinstance(distribution, CategoricalDistribution):
+        info_distribution = {
+            "is_categorical": True,
+            "distribution": distribution,
+        }
+    elif isinstance(
+        distribution,
+        (
+            DiscreteUniformDistribution,
+            IntLogUniformDistribution,
+            IntUniformDistribution,
+            LogUniformDistribution,
+            UniformDistribution,
+        ),
+    ):
+        info_distribution = {
+            "is_categorical": False,
+            "low": distribution.low,
+            "high": distribution.high,
+        }
+
+    return info_distribution
+
+
+def _get_distributions(study: "Study", params: Optional[List[str]]) -> Dict[str, BaseDistribution]:
```

Comment:
This function is becoming somewhat non-trivial. It's a private one but I'm wondering if we shouldn't write unit tests for it to be certain about keeping its behavior as expected.